package.json

```diff
@@ -67,45 +67,46 @@
     "babel-preset-stage-1": "^6.0.0",
     "babel-standalone": "^6.4.4",
     "codemirror": "^5.15.2",
-    "css-loader": "~0.9.0",
     "prop-types": "^15.5.10",
     "react": "^15.4.2",
     "react-addons-test-utils": "^15.4.2",
     "react-codemirror": "^1.0.0",
     "react-dom": "^15.4.2",
     "rimraf": "^2.4.0",
-    "style-loader": "~0.8.0",
-    "url-loader": "~0.5.5",
-    "webpack": "^1.10.0"
+    "webpack": "^1.12.3"
   },
   "devDependencies": {
     "babel-eslint": "^5.0.0",
     "chai": "^3.2.0",
+    "css-loader": "~0.9.0",
     "eslint": "^0.24.1",
     "eslint-config-defaults": "^3.0.3",
     "eslint-plugin-filenames": "^0.1.1",
     "eslint-plugin-react": "^2.6.4",
+    "file-loader": "^0.11.2",
     "isparta-loader": "^2.0.0",
-    "karma": "^0.12.37",
+    "karma": "^0.13.9",
     "karma-chrome-launcher": "^0.2.0",
     "karma-coverage": "^0.4.2",
-    "karma-firefox-launcher": "^0.1.6",
+    "karma-firefox-launcher": "^1.0.1",
     "karma-ie-launcher": "^0.2.0",
     "karma-mocha": "^0.2.0",
-    "karma-phantomjs-launcher": "^0.2.0",
-    "karma-phantomjs-shim": "^1.0.0",
+    "karma-phantomjs-launcher": "^1.0.4",
+    "karma-phantomjs-shim": "^1.1.1",
     "karma-safari-launcher": "^0.1.1",
     "karma-sauce-launcher": "^0.2.14",
     "karma-spec-reporter": "0.0.20",
-    "karma-webpack": "^1.6.0",
+    "karma-webpack": "^1.7.0",
     "lodash": "^4.13.1",
     "mocha": "^2.2.5",
     "opener": "^1.4.1",
-    "phantomjs": "^1.9.17",
+    "phantomjs-prebuilt": "^2.1.15",
```

Comment:
Evidently `yarn` can build `phantomjs-prebuilt` but not `phantomjs`!

Reply:
`phantomjs` is deprecated. `phantomjs-prebuilt` is the modern version.